src/main/java/Functionality.java

```diff
@@ -0,0 +1,93 @@
+import java.util.ArrayList;
+
+public class Functionality {
+
+    private ArrayList<Task> tasks;
+    private String linedivider = "____________________________________________________________\n";
+    public Functionality() {
+        tasks = new ArrayList<>();
+    }
+
+    public void enter(String s) {
+        if (s.equals("list")) {
+            list();
+        } else if (s.length() > 5 && s.substring(0,4).equals("done")) {
```

Comment:
very hard to maintain this method and understand what is the method trying to check
u can give boolean names like boolean isSomeTask = s.length() > 5

Reply:
it can be a good time to start with java docs